### ISSUE TITLE
Dragging a file when there are no pipelines gives no feedback

### ISSUE THREAD

**Issue by astrojuanlu** [orchest/orchest#1009]
**Describe the bug**
I was confused trying to drag a file to the pipeline editor and nothing was happening, to realize that I needed a pipeline first.

**Expected behavior**
The UI gives an error message or tells me what to do.

**To Reproduce**

1. Create a new project
2. Go to the pipeline editor
3. Create a new file
4. Try to drag it

**Screenshots**

https://user-images.githubusercontent.com/316517/171226937-e8bafd22-362a-40ea-bb5f-9d3887416554.mp4

**Environment**

- OS: Linux Mint
- Browser: Firefox 100
- Orchest's version (in the settings page): v2022.05.4

**Comment by astrojuanlu** [comment]
I think this has been discussed before somewhere else, but it would be cool to hand hold the user a bit more with their first pipeline. Some proposed flows:

- When the user creates a project, add a pipeline there. A bit intrusive.
- When the user opens an empty project, a dialog asks "Create a pipeline? Yes/No" With a checkbox "Don't show this again".
- A big call-to-action in the middle of the canvas saying "create a pipeline to start working" or similar

**Comment by yannickperrenet** [comment]
Since we now automatically create a Pipeline when a Project is created (#1261), it is now much less likely that someone will hit this case. In addition, as it stands currently, users have a hard time discovering the ability to drag-and-drop steps. Thus I would say that this issue has low priority.

**Comment by astrojuanlu** [comment]
>  In addition, as it stands currently, users have a hard time discovering the ability to drag-and-drop steps.

I agree with this, are we tracking that somewhere? Would be cool to have a more visible onboarding in that aspect

**Comment by yannickperrenet** [comment]
> are we tracking that somewhere?

We are. It is part of some UX studies that @ncspost is running :raised_hands:
